playground/index.js

```diff
@@ -1,4 +1,4 @@
-const passable = require('passable');
+const passable = require('../dist/Passable.min');
```

Comment:
Maybe even `../dev/Passable`, you'll need to have the dev server running, though

Reply:
I'm not sure it's in this PR's scope, but sounds good.